FacebookImagePicker/OLFacebookAlbumRequest.m

```diff
@@ -20,12 +20,8 @@ @implementation OLFacebookAlbumRequest
 
 + (void)handleFacebookError:(NSError *)error completionHandler:(OLFacebookAlbumRequestHandler)handler {
     NSString *message;
-    if ([FBErrorUtility shouldNotifyUserForError:error]) {
-        message = [FBErrorUtility userMessageForError:error];
-    } else if ([FBErrorUtility errorCategoryForError:error] == FBErrorCategoryAuthenticationReopenSession) {
-        message = @"Your current Facebook session is no longer valid. Please log in again.";
-    } else if ([FBErrorUtility errorCategoryForError:error] == FBErrorCategoryUserCancelled) {
-        message = @"The app requires authorization to access your Facebook photos to continue. Please open Settings and provide access.";
+    if ([error.userInfo objectForKey:@"FBSDKErrorLocalizedDescriptionKey"]) {
+        message = [error.userInfo objectForKey:@"FBSDKErrorLocalizedDescriptionKey"];
```

Comment:
See this page : https://developers.facebook.com/docs/ios/errors (in addition to https://developers.facebook.com/docs/ios/upgrading-4.x)

I didn't get everything (the part about RecoveryAttempting I didn't get at all) but we should be ok with this. Let me know what you think.